src/components/accordion/_accordion.scss

```diff
@@ -202,7 +202,7 @@
     // We also want to avoid styling when the secttion is pressed or `:active`
     // to avoid the different focus styles from flashing quickly while the user interacts with the section.
     .govuk-accordion__section:not(.govuk-accordion__section--expanded) .govuk-accordion__section-button:focus:not(:active) {
-      @include govuk-focusable-text;
+      @include govuk-focusable-fill-css;
```

Comment:
The original mixins assume you always want the state on a regular `:focus` but you can see here It is useful to be able to customise when to show the focus.

At the moment we'd have to have an additional mixin without the `:focus` selector.

You can also see this in the file upload component.